### DISCUSSION TITLE
Hi 4070 12G

### DISCUSSION THREAD

**Discussion by Bernishik** [GpuZelenograd/memtest_vulkan#35] [Card-specific memtest_vulkan error logs]
Hello, I need assistance with my 4070 GPU. It's experiencing random crashes in games after a period of PC(8+ hours in browser) usage. However, after restarting the PC, it works without any crashes in games for some time. So, I find myself needing to restart the PC whenever I want to play games without encountering crashes. I'm wondering if this is a hardware issue. Could you please confirm that problem in gpu?

first test(2024-03-16T15:20:28.113580Z) i started after 8h+ in browser it throws errors

After that, I ran the test several more times, but there were no errors. Then, I ran the test simultaneously with the game, and that's when errors started occurring (2024-03-16T16:35:40.761399Z).

Following this, I restarted the PC and left it for some time. Initially, there were no errors, but after a while, errors started accumulating (last test).

[memtest_vulkan.log](https://github.com/GpuZelenograd/memtest_vulkan/files/14625425/memtest_vulkan.log)

**Comment by galkinvv** [comment]
According to the log the errors are quite systematic. All of the loks like bit 0x18 flip (out of 0x20-bit test unit).

This looks like a hardware issue, but its hard to tell if this is VRAM chip or GPU chip.
However the behaviour you described regarding reboot looks quite strange. As a hypothesis - maybe the fans are too slow and the memory gets too hot during idle... Monitor temps with GPU-Z.

However, your card has very low count of errors. So while it looks like a hardware issue, chances are that it can be workarounded (maybe temporarily until it gets worse...).
If you have no options other then "continue using that GPU" - the generic suggestion for such low errors count would be - "use MSI Afterburner to make fans faster (say ~2500RPM) and all clocks a 10% lower"

**Reply by Bernishik** [reply]
@galkinvv Hello, I sold my old graphics card and bought an RTX 4070 TI Super. I haven't had a chance to test it in games yet, but I ran your test for an extended period of time and I'm still getting errors. Can you confirm if these are the same errors, or if they're different? I'm so unlucky that the second graphics card is defective, or is there some other issue in my system not related to gpu? Please advise. I'm attaching the full log, but you only need to look at the last test.

[memtest_vulkan.log](https://github.com/GpuZelenograd/memtest_vulkan/files/14808691/memtest_vulkan.log)

**Reply by galkinvv** [reply]
They looks different. For older GPU you had the single bit flip issue on a lane 0x18, on quite different addresses.

The log for 4070Ti Super has all errors for adress 0x39?????? and  many of them multi-bit errror, not all of them on single bit 0x18.
So the errors has similarities but still different.

memtest_vulkan typically heats up the memory quite huge. Whats your temps displayed in hwinfo64 during test? Are you sure that you have some air flow inside ypur PC? The GPU of course cools down itslef, but for its cooling to work at least small fresh air income is neede

And of course - check it in games - memtest_vulkan can be heavier load then games.
Overall, maybe 4070 Ti Super is too new and not enough tuned in a driver.

**Reply by Bernishik** [reply]
Hello, the first issue (related to the old video card) is definitely not related to temperature or clocks. I deliberately conducted separate tests, including setting the coolers to maximum and significantly reducing the chip and memory clocks. In such cases, the memory temperature was around 70 degrees, hot spot 60 degrees, and chip 50 degrees and errors still throws. As for the new video card, I'll try to conduct similar tests, but I highly doubt that the problem will be related to overheating since the case ventilation is good.

**Reply by Bernishik** [reply]
just tested new videocard with max fans for 2 hours, seems there are no errors
![image](https://github.com/GpuZelenograd/memtest_vulkan/assets/37352975/2996a8f3-48d2-4212-95e1-54e9eb0e6295)
![image](https://github.com/GpuZelenograd/memtest_vulkan/assets/37352975/2c3ef062-fd38-4b53-83e8-b8a33d0e3f59)

**Reply by galkinvv** [reply]
Great!
I noticed the other moment in the screenshots above VS earlier 4070 Ti Super log. The speed - on the screenshots above the check speed is near 550-570 GB/sec, but on the earlier log with errors - it quickly dropped to 395GB/sec. Looks like thermal throttling or other apps activity

So chances are that something was too hot during the test that gived errors on 4070Ti Super.
Having some errors in test during overheating is not "perfect" behaviour of a hardware but is typical.
